AdobeStockAsset/composer.json

```diff
@@ -1,12 +1,13 @@
 {
     "name": "magento/module-adobe-stock-asset",
-    "description": "N/A",
+    "description": "Magento module responsible for Adobe Stock assets handling on Magento side implementation",
     "require": {
         "php": "~7.1.3||~7.2.0||~7.3.0",
         "magento/framework": "*",
-        "magento/module-user": "*",
         "magento/module-adobe-stock-asset-api": "*",
```

Comment:
I couldn't find `magento/module-adobe-stock-asset-api` mentions in `AdobeStockAsset` module